interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
```

Comment:
.graph is "more generic?  just a general field which can be used for anything?   It would not have to point to a graph object?   Maybe i have missed a deeper discussion